docs/Technical API/Sensor/readme.md

```diff
@@ -0,0 +1,58 @@
+# Sensor class
+
+`Sensor`s are procedures to access external resources and procedures. For example, reading from raw data, feature engineering staffs, and preprocessing procedures.
+"Sensors" are looked at as blackboxes in a program. Users use sensors as callable objects. Underlying `forward()` function will be used to calculate an output. One can override `forward()` function to customize how the sensor get the output.
```

Comment:
First thing can you specify any limitations in the input and output of the sensors?